packages/react-provider/src/hooks/use-hash.ts

```diff
@@ -0,0 +1,20 @@
+import { useMemo } from "react";
+
+function hash(text) {
```

Comment:
Do we still need to attribute source and inspiration for this?

Reply:
like make a comment?

Reply:
yes